src/elfpy/markets.py

```diff
@@ -45,13 +45,15 @@ def __init__(
             init_share_price=1,
         ),
         position_duration: StretchedTime = StretchedTime(365, 1),
-        fee_percent: float = 0,
+        trade_fee_percent: float = 0,
+        redemption_fee_percent: float = 0,
     ):
         # market state variables
         self.time: float = 0  # t: timefrac unit is time normalized to 1 year, i.e. 0.5 = 1/2 year
         self.pricing_model = pricing_model
         self.market_state: MarketState = market_state
-        self.fee_percent: float = fee_percent  # g
+        self.trade_fee_percent: float = trade_fee_percent  # g
+        self.redemption_fee_percent: float = redemption_fee_percent  # g
```

Comment:
this is not `g` -- pls fix or remove the comment

Reply:
its not, neither is trade fee, which is phi.  updated the comments